Documentation/ceph-pool-radosnamespace.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
How about if we don't backport this feature to 1.8 and wait for 1.9? In that case, please add to the feature list in PendingReleaseNotes.md.

Reply:
Yes this is fine i will add this to pendingreleasenotes